test/Mono.Linker.Tests/TestCasesRunner/ResultChecker.cs

```diff
@@ -659,6 +659,13 @@ void VerifyKeptAllTypesAndMembersInAssembly (AssemblyDefinition linked)
 			}
 		}
 
+		bool IsProducedOnlyByAnalyzer (CustomAttribute attr)
```

Comment:
Similarly to the analyzer I would prefer this to work the other way round (test for ProducedBy Linker)

Reply:
In fact it seems the code actually does it already, in which case I would just rename the method and swap the return value meaning. The linker tests should only care about "linker" stuff - and "ignore" anything else.